scripts/jenkins-yoctobuild-build.sh

```diff
@@ -301,7 +301,11 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
                 if ! grep "COPY --from=build" Dockerfile; then
                     gulp build
                 fi
-                docker build -t $docker_url:pr .
+                docker build \
+                    -t $docker_url:pr \
+                    --build-arg GIT_REF=$(git describe) \
+                    --build-arg GIT_COMMIT=$(git rev-parse HEAD) \
```

Comment:
can the `git  rev-parse HEAD` get a short flag? so `git rev-parse --short HEAD` ... I fear this would get a bit messy with the tooltip otherwise 😟

Reply:
Ah, yes, I missed the "SHORT_SHA" reference in the original. I'll fix it.

Reply:
Perfect! Thank you!

Reply:
Fixed!